File: desktop/src/main/java/com/tp/holdem/desktop/DesktopLauncher.java
package com.tp.holdem.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.tp.holdem.core.Holdem;

public class DesktopLauncher {
	public static void main(String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.title = "HoldemTP";
		config.width = 1024;
		config.height = 700;
		
		new LwjglApplication(new Holdem(), config);
	}
}

File: desktop/src/main/kotlin/com/tp/holdem/desktop/DesktopLauncher.kt
package com.tp.holdem.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.tp.holdem.client.Holdem

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.title = "HoldemTP"
        config.width = 1024
        config.height = 700

        LwjglApplication(Holdem(), config)
    }
}
